apps/landingPage/pages/_document.js

```diff
@@ -1,9 +1,18 @@
 // @flow
 
 import React from 'react';
-import Document from 'next/document';
+import Document, { Head, Main, NextScript } from 'next/document';
 import { ServerStyleSheet } from 'styled-components';
 
+const globalStyle = {
```

Comment:
any reason we cannot use [createGlobalStyle](https://www.styled-components.com/docs/api#createglobalstyle)?

Reply:
I guess not, seem to work fine with it.